src/types.ts

```diff
@@ -1,10 +1,18 @@
-import {FileCollectionOptions} from "./utils/mochaCliExports.js";
+export interface FileCollectionOptions {
+  /** File extensions to use */
+  extension: string[];
+  /** Files, dirs, globs to ignore */
+  ignore: string[];
+  /** Find files recursively */
+  recursive: boolean;
+  /** Glob pattern to load spec */
+  spec: string[];
+}
 
-export type Opts = Partial<FileCollectionOptions> & {
+export type StorageOptions = {
   defaultBranch?: string;
   persistBranches?: string[];
   benchmarksPerBranch?: number;
-  threshold: number;
```

Comment:
I see this was moved to `benchmarkOptions` in CLI options and is exported below.  Why was it exported on both types before but no longer needs to be?

Reply:
All benchmark options are also CLI options except `only`, `skip` and `noThreshold`.